aws-production-2/main.tf

```diff
@@ -103,9 +136,16 @@ module "aws_asg_org" {
   worker_asg_scale_in_threshold = 16
   worker_asg_scale_out_qty = 2
   worker_asg_scale_out_threshold = 8
+  worker_cache_bucket = "${var.worker_org_cache_bucket}"
   worker_config = <<EOF
-${file("${path.module}/config/worker-env-org")}
+### ${path.module}/config/worker-org-local.env
+${file("${path.module}/config/worker-org-local.env")}
+### ${path.module}/config/worker-org.env
+${file("${path.module}/config/worker-org.env")}
+### ${path.module}/worker.env
 ${file("${path.module}/worker.env")}
+
+export TRAVIS_WORKER_AMQP_URI=${module.rabbitmq_worker_config_org.uri}
 EOF
   worker_docker_image_android = "quay.io/travisci/ci-amethyst:packer-1473386113"
```

Comment:
No change to anything about how we use quay happening here :smile_cat:, promise!